### PR TITLE
Change how stage edit page allows user to select deploy_groups

### PR DESCRIPTION
**Pull request by zendesk-shender**
Hopefully making things a bit easier to select deploy groups for a stage.
apr 02 2015 14 01

/cc @zendesk/runway @zendesk/samson

### References
 - Jira link: https://zendesk.atlassian.net/browse/RUN-66

### Risks
 - None

### CODE REVIEW

**Comment by gabetax** [comment]
[indeterminate checkboxes](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate) could be a nice polish on the top levels, but not worth blocking on.

File: app/views/stages/_fields.html.erb [hunk 1]
```diff
@@ -26,19 +26,54 @@
...
+              </tr>
+            </thead>
+            <tbody>
+              <% @environments.map {|e| e.deploy_groups.count }.max.to_i.times do |index| %>
+                <tr>
+                  <% @environments.each do |environment| %>
+                    <% if environment.deploy_groups[index] %>
+                      <td>
+                        <label>
+                          <input
```
Comment (gabetax) [inline app/views/stages/_fields.html.erb]: I'd be inclined to use the FormTagHelper methods if straight up FormHelper is too complicated in this scenario, but the only real win is consistency I suppose.

**Comment by gabetax** [comment]
:+1: big UI improvement

**Comment by zendesk-shender** [comment]
Thanks @gabetax for the pointer to the indeterminate checkbox state. That should work now as seen in the gif above.
I also changed the HTML to use the form tag helpers as much as possible as well.

**Pull request merged**